apps/redis.go

```diff
@@ -17,8 +17,49 @@ package apps
 import (
 	"github.com/GoogleCloudPlatform/ops-agent/confgenerator"
 	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/fluentbit"
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/otel"
 )
 
+type MetricsReceiverRedis struct {
+	confgenerator.ConfigComponent `yaml:",inline"`
+
+	confgenerator.MetricsReceiverShared `yaml:",inline"`
+
+	Endpoint string `yaml:"endpoint" validate:"omitempty,url"`
+	Password string `yaml:"password" validate:"omitempty"`
```

Comment:
It doesn't look like you're using the password parameter anywhere?

Reply:
Added to otel config and updated the golden confs

Reply:
Thanks; I assume that explicitly specifying it in the config file as an empty string still works as a default?

Can you leave a TODO here with the plan to add username for ACL authentication?
(https://redis.io/topics/security and https://redis.io/commands/auth)

Reply:
Will do

Reply:
Added